src/linux/Packaging.Linux/install-from-source.sh

```diff
@@ -189,7 +189,7 @@ case "$distribution" in
         $sudo_cmd apk update
 
         # Install dotnet/GCM dependencies.
-        install_packages apk add "curl git icu-libs krb5-libs libgcc libintl libssl1.1 libstdc++ zlib which bash coreutils gcompat"
+        install_packages apk add "curl git icu-libs krb5-libs libgcc libintl libssl3 libstdc++ zlib which bash coreutils gcompat"
```

Comment:
According to the .NET docs, the dependency for OpenSSL should still be 1.1 on 3.14.x and older Alpine.. we may need to switch on the Alpine version in this script.

https://learn.microsoft.com/en-us/dotnet/core/install/linux-alpine#dependencies

Reply:
But it builds ;-)